Odata-docs/webapi/unittest-e2etest.md

```diff
@@ -6,30 +6,28 @@ ms.date: 7/1/2019
 # Unit Test and E2E Test
 **Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v7.md)][!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v6.md)]
 
-In OData WebApi, there are unit test, e2e test for V3 and V4, those [test cases](https://github.com/OData/WebApi/tree/master/OData/test) are to ensure the feature and bug fix, also to make sure not break old functionality.
+In OData WebApi, there are unit tests, e2e tests for V3 and V4, those [test cases](https://github.com/OData/WebApi/tree/master/OData/test) are to verify that features and bug fixes work as intended, and also to make sure they do not break existing functionality.
 
-### Unit Test
-Every class in OData WebApi has it's own unit test class, for example:
+### Unit Tests
+Every class in OData WebApi has its own unit test class, for example:
 OData/src/System.Web.OData/OData/Builder/ActionLinkBuilder.cs 's test class is 
-OData/test/UnitTest/System.Web.OData.Test/OData/Builder/ActionLinkBuilderTests.cs.
-
-You can find that the structural under `System.Web.OData` folder and `System.Web.OData.Test` folder are the same, also for V3 `System.Web.Http.OData.Test`, so if your pull request contains any class add/change, you should add/change(this change here means add test cases) unit test file.
+OData/test/UnitTest/System.Web.OData.Test/OData/Builder/ActionLinkBuilderTests.csYou can find that the structures under the `System.Web.OData` folder and the`System.Web.OData.Test` folder are the same, and the same goes for V3 `System.Web.Http.OData.Test`. So if your pull request contains any class addition/change, you should add/change(here "change" means adding test cases) the corresponding unit test file.
```

Comment:
Formatting changed?

Reply:
Thanks for pointing that out. I've made the change.